github/repos_collaborators_test.go

```diff
@@ -162,7 +162,7 @@ func TestRepositoriesService_AddCollaborator(t *testing.T) {
 		w.WriteHeader(http.StatusNoContent)
 	})
 
-	_, err := client.Repositories.AddCollaborator(context.Background(), "o", "r", "u", opt)
+	_, _, err := client.Repositories.AddCollaborator(context.Background(), "o", "r", "u", opt)
```

Comment:
Please show an example of the returned structure in the test, as you see other tests doing.